### ISSUE TITLE
Unable to exit NVDA while Welcome to NVDA dialog is open

### ISSUE THREAD

**Issue by robertspangler** [nvaccess/nvda#6176]
Hello, I noticed that, when attempting to exit NVDA while the new user dialog/Welcome to NVDA dialog is open, the program will not close.  I tried it with the exit options dialog enabled and disabled.  When it is enabled, if you choose Exit from the list and hit OK, the program will not close if the welcome dialog is open.  If the exit dialog is disabled, pressing exit/insert+Q simply does nothing if the welcome dialog is open.  I thought that perhaps the program simply can't close if any of its dialogs are open but this isn't the case; I tested by opening the general preferences dialog and NVDA did exit with that dialog open.  Minor issue, but thought I would point it out.

**Comment by josephsl** [comment]
Hi,

Confirmed.

Technical: Hmmm, welcome dialog is a modal dialog. This means other parts of NVDA will be blocked by this dialog, including ability to exit NVDA.

@jcsteh and others: do you think it's possible to force NVDA to quit despite a modal dialog being opened? Thanks.

**Comment by robertspangler** [comment]
Why must this window be modal?  I understand that it is a useful tool for
new users, but I don't see why it should block other parts of NVDA.  Can't
it be set up like the preferences dialogs?

**Comment by jcsteh** [comment]
Duplicate of #1740.
